webapps/ui/cockpit/client/scripts/components/callActivityOverlays/template.html

```diff
@@ -1,5 +1,15 @@
-<div class="activity-top-right-position">
-  <button type="button" role="button" class="btn btn-sm btn-default call-btn">
-    <span class="glyphicon glyphicon glyphicon-link"></span>
-  </button>
-</div>
+<button type="button" role="button" class="bjs-drilldown">
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 24 24"
+    width="20"
+    height="20"
+    style="color: white"
+  >
+    <path
+      fill-rule="evenodd"
+      fill="currentColor"
+      d="M13.253 10.406c.33.348.33.894 0 1.242a.842.842 0 01-1.19 0 1.233 1.233 0 00-1.796 0l-2.704 2.82a1.368 1.368 0 000 1.877 1.233 1.233 0 001.797 0l1.6-1.671a.205.205 0 01.23-.049c.428.183.89.276 1.355.274h.085a.216.216 0 01.213.22.226.226 0 01-.062.158l-2.224 2.321a2.884 2.884 0 01-2.094.902c-1.638-.001-2.964-1.388-2.963-3.096 0-.82.312-1.605.866-2.185l2.694-2.813a2.89 2.89 0 012.096-.906 2.915 2.915 0 012.097.906zm4.379-4c1.157 1.208 1.157 3.167 0 4.376l-2.695 2.814c-.374.393-.849.674-1.373.813h-.024l-.122.027-.068.013-.146.022h-.071c-.056 0-.104.011-.151.014l-.127.015h-.16a2.824 2.824 0 01-.316-.031 3.304 3.304 0 01-.256-.055l-.125-.036a1.631 1.631 0 01-.136-.046c-.044-.018-.09-.033-.135-.053-.044-.02-.089-.038-.133-.06a2.94 2.94 0 01-.847-.618.907.907 0 010-1.247.841.841 0 011.19 0 1.232 1.232 0 001.794 0l.695-.721.017-.02 1.985-2.07a1.37 1.37 0 000-1.877 1.233 1.233 0 00-1.797 0l-1.593 1.662a.205.205 0 01-.23.049 3.424 3.424 0 00-1.354-.277h-.078a.212.212 0 01-.2-.135.228.228 0 01.045-.243l2.22-2.316a2.875 2.875 0 014.19 0z"
```

Comment:
Can we move the icon into a dedicated file? If we bundle all our icons whenever we use them, it blows up our Cockpit `plugin.js` quite much.

Reply:
Do we already use this method somewhere? I imagine having it externally and only load it when needed would cause flickering. In networks with high latency, it might take a few ms to fetch the SVG. I'm currently aware of 2 ways we use svgs:

1. Inline it (e.g. https://github.com/camunda/camunda-bpm-platform/blob/48df3ca0d9e796b563ced0a1468bf6da7de84d83/webapps/camunda-commons-ui/lib/widgets/header/cam-widget-header.js#L24)
2. As part of a font (bpmn-font, glyphicons). Those are also loaded at page load and impact load time as well

Reply:
Can we create a custom font? We could gradually add more custom SVGs in the future when necessary.

Reply:
Creating a font seems very impractical to me. With a single svg/inline, we can store it in the component where it is used. Should we want to remove/change it later, it is easily overlooked in a font.
We also have the overhead of creating and maintaining the actual font files (`.woff`, `.tff`, etc.) every time we want to use a new SVG. 

Instead of a font, we could use a `:before` element which has the svg set as a background image. This way we maintain a singular SVG file with a similar semantic to a font

Reply:
An SVG sprite. Sounds feasible!

Reply:
Lets do that then :+1: